app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1963,4 +1974,54 @@ private void searchFiles() {
   public boolean checkNull(View view) {
     return view != null;
   }
+  public static void sendNotification(String message,Context context){
+
+        Intent actionIntent_stop = new Intent(context, NotificationService.class)
+                .setAction(ACTION_STOP);
+        PendingIntent actionPendingIntent_stop = PendingIntent.getService(context, 0,
+                actionIntent_stop, PendingIntent.FLAG_ONE_SHOT);
+        Intent actionIntent_pause_play = new Intent(context, NotificationService.class)
+                .setAction(ACTION_PAUSE);
+        PendingIntent actionPendingIntent_pause_play = PendingIntent.getService(context, 0,
+                actionIntent_pause_play, PendingIntent.FLAG_ONE_SHOT);
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
+                .setSmallIcon(R.drawable.kiwix_icon_with_title)
+                .setContentTitle("Read Aloud")
```

Comment:
Don't use hard-coded strings. Check if ```Read Aloud``` is available in the strings.xml. If not, you can declare it over there.

Reply:
Actually, this method is defined as static which enables it to be called by KiwixTextToSpeech.java class for notifications. Hence getString() which is a non-static method cannot be called by a static context.

Reply:
Context may help. Solutions for such cases are available on Stack overflow.

Reply:
The only solution I found was to remove static from the function definition but removing static won't allow KiwixTextToSpeech to call the sendNotification function.

Reply:
Pass in a ```Context``` (i.e. ```Activity```) instance as a parameter object to static method. Then invoke ```getString``` on the parameter.

Reply:
Yes, I tried that and it worked. Thank you :)